webgui/src/flipmouse/js/tabslot.js

```diff
@@ -35,13 +83,95 @@ tabSlot.initSlots = function () {
     });
 };
 
+tabSlot.initProcesses = function () {
+	if(inElectron === false) {
+		var resultList = ["not available / nicht verfügbar"];
+		//add the process list to the GUI
+		L('.process-select').forEach(function (elem) {
+			elem.innerHTML = L.createSelectItems(resultList);
+		});
+		return;
+	}
+	
+	storage.setDataPath(require('os').homedir());
+	console.log("Looking for config json here: " + storage.getDataPath());
+	
+	//get an active process if one is set for this slot
+	storage.get('FLipMouseGUIConfiguration',function(error, processActivationList) {
+		//if we got a list back (JSON file exists & contains data)
+		if (typeof processActivationList !== 'undefined') {
+			//find active slot in list
+			var index = processActivationList.findIndex(function(item, i){
+				 return item.slot === flip.getCurrentSlot()
+			});
+			activateOnProcess = processActivationList;
```

Comment:
maybe rename `activateOnProcess` to `storedProcessActivationList` or `storedProcessMapping` or similar. I needed to get to this line to understand what `acticateOnProcess` is ;)

Reply:
Is renamed